src/display/api.js

```diff
@@ -2262,9 +2271,11 @@ class WorkerTransport {
       isCompressed: params.cMapPacked,
     });
 
-    this.StandardFontDataFactory = new params.StandardFontDataFactory({
-      baseUrl: params.standardFontDataUrl,
-    });
+    if (!params.standardFontDataWorkerFetch) {
```

Comment:
`this.StandardFontDataFactory` needs to be defined *unconditionally*, please note the comment in the `src/core/evaluator.js` file.